wolfssl/openssl/evp.h

```diff
@@ -372,6 +376,39 @@ struct WOLFSSL_EVP_PKEY_CTX {
     int nbits;
 };
 
+#if defined(WOLFSSL_BASE64_ENCODE) || defined(WOLFSSL_BASE64_DECODE)
+
+#define   BASE64_ENCODE_BLOCK_SIZE  48
+#define   BASE64_ENCODED_BLOCK_SIZE 64
+#define   BASE64_DECODE_BLOCK_SIZE  4
+
+struct WOLFSSL_EVP_ENCODE_CTX {
+    void* heap;
+    int   remaining;     /* num of bytes in data[] */
+    byte  data[128];     /* storage for unprocessed raw data */
```

Comment:
Why was 128 chosen here?  Should we create a define to represent a MAX value instead of using the number directly?

Reply:
Replaced the number with BASE64_ENCODE_BLOCK_SIZE( 48 byte ).